category/models.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+from company.models import Company
+
+
+class Category(models.Model):
+    """A company specific category, to aid in organizing transactions."""
+    label = models.CharField(max_length=128, help_text='Category label')
+    descr = models.CharField(max_length=128, help_text='Category description')
```

Comment:
Why do you shorten description? It's not that much easier to write, and I find it slightly harder to read

Reply:
Also, the help text isn't saying anything more than the class and attribute names do. Maybe explain how `label` and `descr` are different?